config/siodb.conf

```diff
@@ -99,7 +99,14 @@ rest_server.ipv6_https_port = 0
 
 # HTTP chunk size in bytes. Suffixes `k`, `K`, `m`, `M` change units
 # to the kilobytes and megabytes respectively.
-rest_server.chunk_size=64K
+rest_server.chunk_size = 64K
+
+# Maximum buffer size to process request payload to IOMgr. Suffixes `k`, `K`, `m`, `M` change units
+# to the kilobytes and megabytes respectively.
+rest_server.request_payload_buffer_size = 1k
+
+# Read deadline timeout in seconds for connections to IOMgr
+rest_server.iomgr_read_deadline = 5
```

Comment:
I think this should be renamed. "Read deadline" is some specific Go terminology. Please rename to "iomgr_read_timeout".